input/write_reactors.py

```diff
@@ -184,7 +283,7 @@ def region_render(array, template, full_template, output_file):
         os.system('rm ' + country + '_region')
 
 
-def main(csv_file, reactor_template, region_template,
+def main(csv_file, reactor_template, deployinst_template,
```

Comment:
The best option is to aim for a python package that can do both and small scripts which contain the problem specific parts. Let's talk about this. Can you please schedule a meeting with me about it?